moqt-core/src/modules/messages/data_streams/object_datagram.rs

```diff
@@ -56,25 +60,33 @@ impl ObjectDatagram {
     pub fn track_alias(&self) -> u64 {
         self.track_alias
     }
+
+    pub fn group_id(&self) -> u64 {
+        self.group_id
+    }
+
+    pub fn object_id(&self) -> u64 {
+        self.object_id
+    }
 }
 
-impl MOQTPayload for ObjectDatagram {
-    fn depacketize(buf: &mut bytes::BytesMut) -> Result<Self>
+impl DataStreams for ObjectDatagram {
+    fn depacketize(read_cur: &mut std::io::Cursor<&[u8]>) -> Result<Self>
```

Comment:
I thought three methods to realize it.
1. Take all data out to buf, process them, and put it back in the original buf
2. Pass buf itself to the handler, create the cursor in the handler, and read the buf
3. Create the cursor in object_stream_handler in advance and pass it to the handler

In the method 1, since it being written to buf in the separate thread, I have to prepare some processes to keep the order.
The processes in method 2 and 3 are same, and I thought 3 is more simple than 2 because the cursor is generated higher layer.